docs/topics/i18n/translation.txt

```diff
@@ -70,6 +70,16 @@ as a shorter alias, ``_``, to save typing.
        global ``_()`` function causes interference. Explicitly importing
        ``ugettext()`` as ``_()`` avoids this problem.
 
+    .. warning::
```

Comment:
Do you think the wording is okay or should I try to amend to incorporate that idea?